doc/gpo/README.md

```diff
@@ -0,0 +1,22 @@
+# Group Policy Objects
+
+Since version 0.64, PowerToys is released on GitHub with GroupPolicyObject files. You can check these releases on https://github.com/microsoft/PowerToys/releases .
+
+## How to install
+
+- Copy "PowerToys.admx" to "C:\Windows\PolicyDefinitions" or equivalent path if Windows is installed in another path.
```

Comment:
This path is only correct on the local computer. For use within an Active Directory the path has to be `%systemroot%\sysvol\domain\policies\PolicyDefinitions`.

Reply:
We can adopt the docs from Edge in a similar way: https://learn.microsoft.com/en-us/deployedge/configure-microsoft-edge#1-download-and-install-the-microsoft-edge-administrative-template

Reply:
I've done this. Thanks for the tip.